src/OSPSuite.Presentation/Presenters/AbstractExplorerPresenter.cs

```diff
@@ -17,9 +17,31 @@
 
 namespace OSPSuite.Presentation.Presenters
 {
+   public interface ICanDragDropPresenter
```

Comment:
I broke this out from Explorer presenter because I can use the drag/drop binder in my views